docs/content/guides/rows/row-sorting.md

```diff
@@ -453,163 +1184,2251 @@ import 'handsontable/dist/handsontable.full.min.css';
+    hotTableComponentRef.current.hotInstance.rowIndexMapper.moveIndexes(
+      [
+        hotTableComponentRef.current.hotInstance.toVisualRow(0),
+        // you can add more top rows here
+      ], 0);
+
+    hotTableComponentRef.current.hotInstance.rowIndexMapper.moveIndexes(
+      [
+        hotTableComponentRef.current.hotInstance.toVisualRow(4),
+        // you can add more bottom rows here
+      ], 4);
+  };
+
+  return (
+    <HotTable
+      ref={hotTableComponentRef}
+      data={[
+        {
+          brand: 'Jetpulse',
+          model: 'Racing Socks',
+          price: 30,
+          sellDate: '11/10/2023',
+          sellTime: '01:23',
+          inStock: false,
+        },
+        {
+          brand: 'Gigabox',
+          model: 'HL Mountain Frame',
+          price: 1890.90,
+          sellDate: '03/05/2023',
+          sellTime: '11:27',
+          inStock: false,
+        },
+        {
+          brand: 'Camido',
+          model: 'Cycling Cap',
+          price: 130.10,
+          sellDate: '27/03/2023',
+          sellTime: '03:17',
+          inStock: true,
+        },
+        {
+          brand: 'Chatterpoint',
+          model: 'Road Tire Tube',
+          price: 59,
+          sellDate: '28/08/2023',
+          sellTime: '08:01',
+          inStock: true,
+        },
+        {
+          brand: 'Eidel',
+          model: 'HL Road Tire',
+          price: 279.99,
+          sellDate: '02/10/2023',
+          sellTime: '13:23',
+          inStock: true,
+        },
+      ]}
+      columns={[
+        {
+          title: 'Brand',
+          type: 'text',
+          data: 'brand',
+        },
+        {
+          title: 'Model',
+          type: 'text',
+          data: 'model',
+        },
+        {
+          title: 'Price',
+          type: 'numeric',
+          data: 'price',
+          numericFormat: {
+            pattern: '$ 0,0.00',
+            culture: 'en-US'
+          },
+          className: 'htLeft',
+        },
+        {
+          title: 'Date',
+          type: 'date',
+          data: 'sellDate',
+          className: 'htRight',
+        },
+        {
+          title: 'Time',
+          type: 'time',
+          data: 'sellTime',
+          correctFormat: true,
+          className: 'htRight',
+        },
+        {
+          title: 'In stock',
+          type: 'checkbox',
+          data: 'inStock',
+          className: 'htCenter',
+        },
+      ]}
+      // enable sorting for every column
+      columnSorting={true}
+      // exclude rows number 1 and 5 from sorting
+      afterColumnSort={exclude}
+      height="auto"
+      width="auto"
+      licenseKey="non-commercial-and-evaluation"
+    />
+  );
+};
+
+/* start:skip-in-preview */
+ReactDOM.render(<HandsontableComponent />, document.getElementById('example9'));
+/* end:skip-in-preview */
+```
+:::
+
+:::
+
+You can also exclude [frozen rows](@/guides/rows/row-freezing.md) from sorting. In the following demo,
```

Comment:
> Instead of providing an exact number of rows excluded from sorting, you can exclude all frozen rows at once. This is helpful if end users can dynamically adjust the number of frozen rows in your application.